AppCenter/AppCenter.xcodeproj/project.pbxproj

```diff
@@ -230,6 +230,12 @@
 		6E48A5A41D3831FE006E8B5F /* MSChannelUnitConfigurationTests.m in Sources */ = {isa = PBXBuildFile; fileRef = 6E48A5A31D3831FE006E8B5F /* MSChannelUnitConfigurationTests.m */; };
 		6E48A5A71D383893006E8B5F /* MSChannelGroupDefaultTests.m in Sources */ = {isa = PBXBuildFile; fileRef = 6E48A5A61D383893006E8B5F /* MSChannelGroupDefaultTests.m */; };
 		6EB1F40E1D2443B7005F9F99 /* MSChannelUnitDefaultTests.m in Sources */ = {isa = PBXBuildFile; fileRef = 6EB1F40D1D2443B7005F9F99 /* MSChannelUnitDefaultTests.m */; };
+		7839727423B39C2A00620142 /* MSAlertController.h in Headers */ = {isa = PBXBuildFile; fileRef = 78686FAB23B390C500CDA4C1 /* MSAlertController.h */; };
```

Comment:
Why did this move? Isn't it only used in Distribute?

Reply:
I guess it is because alert controller is a kind of utility that could be used for other modules even though it is only used by Distribute at the moment. I don't have any strong opinion on this one as it could stay in Distribute or it could be moved to core for any of upcoming usages. The only concern is that I couldn't think of any future usage of this one. 😅

Reply:
I assume this implementation should be in Core of AppCenter. Possible we may need to display the alert in other modules. And this implementation probably does not have an unambiguous relationship to the distribute module.

Reply:
It's just I do not want to bloat core size for something only used by one module unless there is some technical/logical good reason. I don't see one at this time.

Reply:
I moved this controller to Distribute